rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -639,3 +639,34 @@ def _get_address_set(self, set_name):
 
         ovn_nbctl = self._get_ovn_controller(self.install_method)
         return ovn_nbctl.get("Address_Set", set_name, 'addresses')
+
+    def runCmd(self, ssh, cmd="perf record", process="ovn-controller"):
```

Comment:
This seems very `perf` specific. Also, what if there are multiple processes with name `process` running at the same time. `pidof` will only select one.

Also, why do we need a special treatment for "stop" commands? Can't we have something like below?
``runCmd(ssh, 'perf record -p `pidof ovn-controller`')`` and later ``runCmd(ssh, 'kill `pidof ovn-controller`)''

Reply:
> This seems very `perf` specific. Also, what if there are multiple processes with name `process` running at the same time. `pidof` will only select one.

@dceara why 'perf' specific? at the end 'perf' it is just used as default cmd value, right?
Yes, in theory we can have multiple process in the same sandbox but does it worth to complicate the code now for an out of scope case?

> 
> Also, why do we need a special treatment for "stop" commands? Can't we have something like below?
> `` runCmd(ssh, 'perf record -p `pidof ovn-controller`') `` and later ``runCmd(ssh, 'kill `pidof ovn-controller`)''

@dceara yes, I think it works, I will update it